sdks/go/alpha.go

```diff
@@ -239,21 +240,21 @@ func (a *Alpha) GetListValues(key string) ([]string, error) {
 // AppendListValue appends a string to a List's values list, given the List's key (name)
 // and the string value. Will error if the string already exists in the list.
 // Will error if the key was not predefined in the GameServer resource on creation.
-func (a *Alpha) AppendListValue(key, value string) (bool, error) {
+func (a *Alpha) AppendListValue(key, value string) error {
 	_, err := a.client.AddListValue(context.Background(), &alpha.AddListValueRequest{Name: key, Value: value})
 	if err != nil {
-		return false, errors.Wrapf(err, "could not get List %s", key)
+		return errors.Wrapf(err, "could not get List %s", key)
 	}
-	return true, nil
+	return nil
```

Comment:
```suggestion
	return err
```

Just for consistency with the other methods.

Reply:
yeah, I noticed we usually `return err` at the end of functions where we've already checked that it's nil. Seems a bit redundant?? What do you think about consistently using `return nil` in these cases for the rest of our functions?  I’m happy to make those updates if you agree!

Will push a new simple-game-server image to the prod once the modifications are finalized.

@markmandel please share your thoughts...

Reply:
Actually, for consistency and readability, the last line on all should be `return nil` - doing a `return err` after we've done a `if err != nil` check requires some logical parsing of the code, whereas `return nil` is explicit in what it's doing, and can't really go wrong.

So I'd say, let's change 'em all to `return nil` on the last line.

Reply:
I've updated the code to `return nil` after the `if err != nil` check.

Reply:
Could you give the changes a quick look to make sure they're good to go? Once confirmed, I'll push the new image to prod.

Reply:
LGTM